### DISCUSSION TITLE
crowdsec/appsec error:

### DISCUSSION THREAD

**Discussion by ne0ark** [ZoeyVid/NPMplus#1118] [Q&A] [answered]
I am running into issue where i get following error:

```
time="2024-09-25T12:27:36-04:00" level=info msg="127.0.0.1 - [Wed, 25 Sep 2024 12:27:36 EDT] \"HEAD /v1/decisions/stream HTTP/1.1 403 220.127µs \"Go-http-client/1.1\" \""
time="2024-09-25T12:27:36-04:00" level=error msg="Unauthorized request from '172.17.0.1:43984' (real IP = <external IP>)" name=appsec type=appsec
time="2024-09-25T12:27:36-04:00" level=error msg="while fetching bouncer info: ent: bouncer not found" ip=172.17.0.1
```

I am not sure why I am getting error for bouncer info: ent: counter not found. Also I am getting blocked when using internal ip.

**Comment by Zoey2936** [comment]
can you send me your corwdsec conf file and both compose files?

**Comment by ne0ark** [comment]


**Reply by Zoey2936** [reply]
compose files and crowdsec.conf?

**Reply by ne0ark** [reply, marked as answer]
The issue is resolved. The bouncer was not correctly registered. Thanks

**Reply by Zoey2936** [reply]
ok good!

**Reply by DKT69** [reply]
> The issue is resolved. The bouncer was not correctly registered. Thanks

how do you fix? im stuck after place sudo cscli bouncers add npm-proxy too.

please help thanks

**Reply by Zoey2936** [reply]
can you open a new discussion for this please and describe the error, please?

**Reply by DKT69** [reply]
im getting same error from the owner post.

**Reply by Zoey2936** [reply]
new discussion please...
